playwright/e2e/crypto/crypto.spec.ts

```diff
@@ -468,6 +469,14 @@ test.describe("Cryptography", function () {
 
             /* log out, and back in */
             await logOutOfElement(page);
+            // Reload to work around a Rust crypto bug where it can hold onto the indexeddb even after logout
+            // https://github.com/element-hq/element-web/issues/25779
+            await page.addInitScript(() => {
+                // Clear localStorage as the user/pageWithCredentials fixtures will re-inject them on reload otherwise
+                // and by logging out we just invalidated the token that will be injected, causing a 401 unauth error.
```

Comment:
The init script is going to re-inject the creds whether or not we do this; the point is that we need to undo that.

```suggestion
                // When we reload, the initScript created by the `user`/`pageWithCredentials` fixtures
                // will re-inject the original credentials into localStorage, which we don't want.
                // To work around, we add a second initScript which will clear localStorage again.
```